contracts/README.md

```diff
@@ -49,8 +55,9 @@ forge install risc0/risc0-ethereum
 [template]: https://github.com/risc0/bonsai-foundry-template
 [Foundry]: https://book.getfoundry.sh/
 [foundry-dependencies]: https://book.getfoundry.sh/projects/dependencies
-[`IRiscZeroVerifier`]: ./src/IRiscZeroVerifier.sol
-[`RiscZeroGroth16Verifier`]: ./src/groth16/RiscZeroGroth16Verifier.sol
-[`RiscZeroVerifierEmergencyStop`]: ./src/RiscZeroVerifierEmergencyStop.sol
-[`RiscZeroVerifierRouter`]: ./src/RiscZeroVerifierRouter.sol
 [groth16-article]: https://www.risczero.com/news/on-chain-verification
+[IRiscZeroVerifier]: ./src/IRiscZeroVerifier.sol
```

Comment:
question: mb this should point to a tagged release?

Reply:
It points locally because they are viewing this on a given git ref. If they are on `main`, they'll expect this link to take them to the implementation on `main`, and if you are interested in the released version they should open this README on that branch instead.

Reply:
When/if we link to this file from the docs, we should link toa tagged release.